rust-toolchain.toml

```diff
@@ -0,0 +1,7 @@
+[toolchain]
+# pin to a specific version of the Rust toolchain
+# so that performance more consistent consistent
+# clarifies how rust influences parameters
+channel = "1.80.0"
+components = ["rust-analyzer", "tree"]
```

Comment:
I'd rather not have a rust-toolchain.toml file to manage

Reply:
benchamarking are scientific measurments? if yes they must be reproducible and use same base line. 
using random rust reduces quality of benchmarks.
canonical latest stable is good target to measure and remeasure.